### PR TITLE
feat(chart): enable creation of scatterplots by using `@xcolumn,@ycolumn` syntax

### PR DESCRIPTION
**Pull request by trieloff**

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -265,9 +265,24 @@ export function chartify(results, description, requestParams) {
...
+            key: String.fromCharCode(120 + i),
+            column: e.substring(1),
+          }))
+          .reduce((acc, { key, column }) => {
+            // eslint-disable-next-line no-param-reassign
+            acc[key] = entry[column];
+            return acc;
+          }, {}))
+        // if it is a single value, then just return that value
+        .map((entry) => (Object.keys(entry).length === 1 ? entry.x : entry));
```
Comment (dbrrt) [inline src/util.js:285]:
```suggestion
        .map((entry) => (Object.keys(entry).length === 1 && entry.hasOwnProperty('x') ? entry.x : entry));
```

**Review by dbrrt — APPROVED** [review]
LGTM!

File: src/util.js (hunk 1 above)
Comment (trieloff) [reply src/util.js]: @dbrrt eslint hated this suggestion https://app.circleci.com/pipelines/github/adobe/helix-run-query/3482/workflows/f6ddaeb9-7920-4389-8b07-9acc6f4a349a/jobs/5870?invite=true#step-105-65_34

**Pull request merged**